remove code that requires functions still under LIBRESSL_NEXT_API

Index: piv-certs.c
--- piv-certs.c.orig
+++ piv-certs.c
@@ -2960,6 +2960,7 @@ out:
 	return (err);
 }
 
+#if 0
 errf_t *
 agent_sign_crl(int fd, struct sshkey *pubkey, X509_CRL *crl)
 {
@@ -3176,6 +3177,7 @@ out:
 	freezero(sig, siglen);
 	return (err);
 }
+#endif
 
 errf_t *
 piv_sign_cert_req(struct piv_token *tkn, struct piv_slot *slot,
